src/services/TechRecordsService.ts

```diff
@@ -215,13 +215,13 @@ class TechRecordsService {
       });
   }
 
-  private createAndArchiveTechRecord(techRecord: ITechRecordWrapper, msUserDetails: any) {
+  public createAndArchiveTechRecord(techRecord: ITechRecordWrapper, msUserDetails: any, newStatusCode: any) {
     const {statusCode} = techRecord.techRecord[0];
     delete techRecord.techRecord[0].statusCode;
+    const isPayloadValid = validatePayload(techRecord.techRecord[0]);
     if (statusCode === STATUS.ARCHIVED) {
       return Promise.reject({statusCode: 400, body: ERRORS.CannotUpdateArchivedRecord});
     }
-    const isPayloadValid = validatePayload(techRecord.techRecord[0]);
```

Comment:
I think this should be placed back where it was, since I think it makes more sense to actually validate the payload only if the record isn't archived, because you're not allowed to update an archived record.